hi-ml-histopathology/src/histopathology/configs/classification/DeepSMILEPanda.py

```diff
@@ -128,7 +136,14 @@ def create_model(self) -> DeepMILModule:
         # no-op IdentityEncoder to be used inside the model
         self.slide_dataset = self.get_slide_dataset()
         self.level = 1
-        return DeepMILModule(encoder=IdentityEncoder(input_dim=(self.encoder.num_encoding,)),
+        self.class_names = ["ISUP 0", "ISUP 1", "ISUP 2", "ISUP 3", "ISUP 4", "ISUP 5"]
```

Comment:
I think this should be an attribute of the dataset so that you could do ` class_names=self..data_module.train_dataset.CLASS_NAMES` as you currently do for the number of classes. This way we don't need to harcode these strings in multiple configs. Optionally, number of classes could also be len(class_names) to avoid any inconsistency